Doc/library/asyncio-stream.rst

```diff
@@ -160,11 +200,122 @@ and work with streams:
 
    .. deprecated-removed:: 3.8 3.10
 
-      `start_unix_server()` is deprecated in favor of `UnixStreamServer()`.
+      `start_unix_server()` is deprecated in favor of :class:`UnixStreamServer`.
 
 
 ---------
 
+StreamServer
+============
+
+.. class:: StreamServer(client_connected_cb, /, host=None, port=None, *, \
+                        limit=2**16, family=socket.AF_UNSPEC, \
+                        flags=socket.AI_PASSIVE, sock=None, backlog=100, \
+                        ssl=None, reuse_address=None, reuse_port=None, \
+                        ssl_handshake_timeout=None, shutdown_timeout=60)
```

Comment:
arguments should be described here. A link to `loop.create_server()` can be used to don't repeat `family`, `flags` etc.

Reply:
Added doc for client_connected_cb and limit. I have added a note that rest of the arguments are passed to `loop.create_server`.